GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kroxylicious Governance
+
+This document outlines the governance model for Kroxylicious.
+This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+[cfc]: https://www.commonhaus.org/bylaws/cf-council.html
+[cdm]: https://www.commonhaus.org/bylaws/decision-making.html
+[ctp]: https://www.commonhaus.org/policies/trademark-policy/
+[coc]: ./CODE_OF_CONDUCT.md
+[contrib]: ./CONTRIBUTING.md
+
+## Table of Contents
+
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+- [Trademark Policy](#trademark-policy)
+- [Contributing](#contributing)
+
+## Roles and Responsibilities
+
+- **Code owners:** Individuals with write access to the repository, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in each repository (`.github/CODEOWNERS`).
+- **Contributors:** Anyone who contributes to the project in any form.
+
+Reference: [About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+
+## Decision Making
+
+This project follows The [Commonhaus decision making][cdm] process.
+
+1. **Consensus-seeking (lazy consensus):** The project primarily aims for a consensus-based decision-making process, where code owners and active contributors discuss and come to an agreement.
+2. **Voting:** In situations where consensus cannot be reached, decisions may be made through a simple majority vote among the code owners.
+3. **Conflict Resolution:** If conflicts arise, code owners are responsible for facilitating a resolution. The [Commonhaus Foundation Council][cfc] (CFC) can be asked to mediate the discussion, if necessary.
```

Comment:
Should we remove this CFC bit? Currently it would be misleading. Should we have a disclaimer somewhere in the doc saying "while we use the Commonhaus process, the project is not yet part of any foundation"

Reply:
Yeah I think you are right @robobario , we could add it back once we are accepted if we think its important.

Reply:
I've commented out the sections related to Commonhaus.